src/index.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  Pocket,
+  Configuration,
+  RelayResponse,
+  RpcErrorResponse,
+  RelayPayload,
+  RelayHeaders,
+  RelayRequest,
+  typeGuard
+} from "pocket-js-core"
+import { TransactionSigner } from "./transaction-signer"
+/**
+ * Pocket Web3 Provider
+ * Sends Relays to a service node in the Pocket Network
+ * @param {string} activeBlockchain - Blockchain hash.
+ * @param {Configuration} configuration - Pocket Configuration.
+ * @param {Configuration} optionalConfiguration - (optional)  Pocket alternate Configuration.
+ * @param {TransactionSigner} transactionSigner - (optional) Transaction signer object.
+ */
+export class PocketProvider {
+  public readonly pocket: Pocket
+  public readonly transactionSigner?: TransactionSigner
+  public activeBlockchain: string
+  public isConnected = true
+
+  constructor(
+    activeBlockchain: string,
+    configuration: Configuration,
+    optionalConfiguration?: Configuration,
+    transactionSigner?: TransactionSigner
+  ) {
+    this.pocket = new Pocket(configuration, optionalConfiguration)
+    this.activeBlockchain = activeBlockchain
+    this.transactionSigner = transactionSigner
+  }
+  /**
+   *
+   * @method send
+   * @param {RelayRequest} relay - Object containing the relay request.
+   * @returns {RelayResponse} - Relay response object.
+   * @memberof PocketProvider
+   */
+  public async send(relay: RelayRequest): Promise<RelayResponse | RpcErrorResponse> {
```

Comment:
This interface is wrong, the type that the `send` function expects is not `RelayRequest` is the ETH JSON RPC request.